CMakeLists.txt

```diff
@@ -11,8 +11,10 @@ set(BIN_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
 set(OEM_THEME_DIR "" CACHE STRING "Define directory containing a custom theme")
 if ( EXISTS ${OEM_THEME_DIR}/OEM.cmake )
      include ( ${OEM_THEME_DIR}/OEM.cmake )
+     set(IS_BRANDED ON)
```

Comment:
This assumption is actually wrong IIRC. From memory nowadays the brander overwrites NEXTCLOUD.cmake directly. This unfortunately can't rely on the presence of OEM.cmake to know it's a branded build. Either brander needs to be changed to go back to using OEM.cmake or OEM.cmake use should be obliterated from the cmake build system here.

None of this should be done by this PR of course. Just stating it because that means this PR won't work as expected with the current brander / client combination and lead to `Theme::isBranded()` giving false positives.